app-ios/Sources/SearchFeature/DayFilterSheetView.swift

```diff
@@ -0,0 +1,88 @@
+import appioscombined
+import Assets
+import Model
+import SwiftUI
+import Theme
+
+private extension DateFormatter {
+    static let eventDateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.calendar = Calendar(identifier: .gregorian)
+        formatter.dateStyle = .long
+        return formatter
+    }()
+}
+
+private extension DroidKaigi2022Day {
+    // NOTE: This implementation is not good, but it is acceptable for this use case.
+    func eventDate() -> String {
+        let eventDateFormatter = DateFormatter.eventDateFormatter
+        let languageCode = Locale.current.language.languageCode?.identifier
+        guard let languageCode = languageCode else { return name }
+        if languageCode == "ja" {
+            eventDateFormatter.dateFormat = "YYYY年MM月d日"
```

Comment:
Please prevent hard code `dateFormat` 🙏🏼 
Can you use template? ref: https://qiita.com/rinov/items/bff12e9ea1251e895306#2-%E6%97%A5%E4%BB%98%E6%96%87%E5%AD%97%E5%88%97%E3%81%B8%E5%A4%89%E6%8F%9B%E6%99%82%E3%81%AB%E3%83%AD%E3%83%BC%E3%82%AB%E3%83%A9%E3%82%A4%E3%82%BA%E3%81%95%E3%82%8C%E3%82%8B%E3%81%B9%E3%81%8D%E7%AE%87%E6%89%80%E3%82%92%E3%83%8F%E3%83%BC%E3%83%89%E3%82%B3%E3%83%BC%E3%83%89%E3%81%97%E3%81%A6%E3%81%84%E3%82%8B

Reply:
fixed in [b036a45](https://github.com/DroidKaigi/conference-app-2022/pull/815/commits/b036a452ebeff76c8081698fb0a52c14fbec9000)